fedimint-server/src/config/api.rs

```diff
@@ -235,8 +235,10 @@ impl ConfigGenApi {
     /// Writes the configs to disk after they are generated
     fn write_configs(&self, config: &ServerConfig, state: &ConfigGenState) -> ApiResult<()> {
         let auth = config.private.api_auth.0.clone();
-        write_new(self.data_dir.join(SALT_FILE), random_salt())
-            .map_err(|e| ApiError::server_error(format!("Unable to write to data dir {e:?}")))?;
+        let io_error = |e| ApiError::server_error(format!("Unable to write to data dir {e:?}"));
+        // TODO: Make writing password optional
+        write_new(self.data_dir.join(PLAINTEXT_PASSWORD), &auth).map_err(io_error)?;
```

Comment:
Should we track this in an issue or will it be a follow-up soonish?

Reply:
Yes I'll create an issue